dashboard/src/components/SearchBox.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { withStyles } from "material-ui/styles";
+// import Typography from "material-ui/Typography";
+import filterIcon from "material-ui-icons/FilterList";
+
+const styles = theme => ({
+  box: {
+    display: "flex",
+    border: "1px solid",
+    borderColor: theme.palette.divider,
+    backgroundColor: "white",
+    boxShadow:
+      "0 0 2px 0 rgba(0,0,0,0.14), 0 2px 2px 0 rgba(0,0,0,0.12), 0 1px 3px 0 rgba(0,0,0,0.20)",
+  },
+  filterIconContainer: {
+    display: "flex",
+    padding: "6px 8px 8px",
+    height: 36,
+    // TODO find proper reference to this in theme, default icon color
+    color: "rgba(0, 0, 0, 0.54)",
```

Comment:
https://github.com/mui-org/material-ui/blob/v1-beta/src/styles/createPalette.js#L32-L37

possibly `theme.palette.action.active` could probably even flip it to `selected` when the input is active.